History.txt

```diff
@@ -1,11 +1,18 @@
 Documentation and download available at https://www.FreeRTOS.org/
 
-Changes after FreeRTOS 202012.00 release
+Changes after FreeRTOS 202012.00-LTS and 202012.01-LTS releases
 
+		+ Update AWS IoT Device Defender and AWS IoT Jobs libraries
+		  to LTS releases.
+		+ Added new OTA library.
         + Added new coreMQTT Agent library.
+			- Refer to https://github.com/FreeRTOS/coreMQTT-Agent-Demos 

```

Comment:
we added basic coreMQTT Agent demo into P3. Other demos are in coreMQTT-Agent_Demos